lib/gnat.ex

```diff
@@ -348,6 +348,14 @@ defmodule Gnat do
     GenServer.call(name, :server_info)
   end
 
+  @doc """
+  Get a randomly generated subscription inbox based on your inbox_prefix.
+  """
+  @spec make_new_inbox(t()) :: binary()
+  def make_new_inbox(pid) do
+    GenServer.call(pid, :make_new_inbox)
```

Comment:
Because this just returns a string with no other IO involved, it seems like this would be more beneficial to not call through a genserver?

https://github.com/nats-io/nats.ex/blob/a01fe4402f5b0d8f7c3db9e4ea45596693bc6baf/lib/gnat.ex#L475-L477

It really is like 2 lines of code for a random new nats-style inbox string.